apps/aestratum/src/aestratum_deploy.erl

```diff
@@ -16,7 +16,7 @@ deploy_payout_contract(#{public := PubKey, secret := PrivKey}) ->
     {ok, WrappedTx}  =
         aect_create_tx:new(#{owner_id    => aeser_id:create(account, PubKey),
                              nonce       => aec_accounts:nonce(Account) + 1,
-                             code        => aect_sophia:serialize(compiled_contract()),
+                             code        => aect_sophia:serialize(compiled_contract(), 2),
```

Comment:
I see this fixes to SOPHIA_CONTRACT_VSN_2 so not to change semantics. Ok.

Reply:
Actually it does change semantics... it was wrong in aestratum_deploy were the Roma version was used to serialize a contract

Reply:
Nope, we were just blind... This is equivalent.